ui/src/views/DashboardView/DashboardView.tsx

```diff
@@ -54,86 +60,87 @@ const DashboardView = () => {
     [catalogussen]
   );
 
-  const filteredZaaktypen = attributeOnQueryFilter(query, zaaktypen, 'omschrijving');
-
+  const filteredZaaktypen = attributeOnQueryFilter(
+    query,
+    zaaktypen,
+    fields as (keyof ZaaktypeT)[]
+  ).map((z) => {
+    const url = z.url ? `/zaaktypen/${uuidExtract(z.url)}` : undefined;
+    return {
+      absolute_url: url,
+      ...z,
+    };
+  });
   const onCatalogusChange = (event: any) => {
     const catalogusUrl = event.target.value;
     localStorage.setItem('catalogus', catalogusUrl);
     setSelectedCatalogus(catalogusUrl);
   };
 
-  const hasError = errorCatalogi || errorZaaktypen;
   const isLoading = loadingZaaktypen || loadingCatalogi;
-  const showNumberOfZaaktypen = !errorZaaktypen && !loadingZaaktypen && catalogussen;
+
+  const selectOptions = [
+    { label: 'Alle catalogussen', value: 'all' },
+    ...(loadingCatalogi ? [] : catalogiOptions).map(([value, label]) => ({ value, label })),
+  ];
 
   return (
-    <Stack component={'article'} spacing={spacings.large} useFlexGap>
-      <Box width={'100%'}>
-        <Typography variant="h1">Dashboard</Typography>
-      </Box>
-      <Stack
-        width={'100%'}
-        direction={'row'}
-        flexWrap={'wrap'}
-        spacing={spacings.medium}
-        useFlexGap
-        position={'sticky'}
-        top={64}
-        zIndex={1}
-        sx={{
-          pb: 2,
-          backgroundColor: 'white',
-          borderBottom: '1px solid',
-          borderColor: 'divider',
+    <>
```

Comment:
Is this one needed? I think everything is inside the `List` tag?

Reply:
I think you're right